API_CHANGES.md

```diff
@@ -3,6 +3,23 @@
 ## Breaking changes
 
 * New feature flag `enableMonitoring` in `org.sonarsource.sonarlint.core.rpc.protocol.backend.initialize.FeatureFlagsDto` allows clients to opt into monitoring with Sentry
+* Removed `org.sonarsource.sonarlint.core.rpc.protocol.SonarLintRpcClient#didRaiseIssue` and associated types. See `raiseIssues` and `raiseHotspots` instead.
```

Comment:
Should be part of the changes for 10.14. Right now it's 10.13.

Reply:
Good catch! Thanks